app/controllers/stacks_controller.rb

```diff
@@ -8,7 +8,7 @@ def new
   def index
     @user_stacks = current_user.stacks_contributed_to
 
-    @stacks = Stack.order('(undeployed_commits_count > 0) desc', tasks_count: :desc)
+    @stacks = Stack.order('(undeployed_commits_count > 0) desc', tasks_count: :desc).to_a
```

Comment:
Because we call `size` twice on `@stacks` in the view. So we might as well fetch it once and for all in the controller.